code/game/objects/items/devices/scanners.dm

```diff
@@ -129,7 +129,7 @@ GENE SCANNER
 
 // Used by the PDA medical scanner too
 /proc/healthscan(mob/user, mob/living/M, mode = 1, advanced = FALSE, to_chat = TRUE)
-	if(isliving(user) && (user.incapacitated() || user.eye_blind))
```

Comment:
Please do not change this logic

Reply:
The new logic is ``if(isliving(user) && (user.incapacitated()))``
This allows blind people to use med scanners.

Reply:
I guess we can cope that they have tts interface or something